language/fr/pages_acp.php

```diff
@@ -79,22 +79,22 @@
+	'ACP_PAGES_FORM_CONTENT_EXPLAIN'	=> 'Le contenu peut être créé en faisant usage des BBCodes normaux de phpBB, les smilies et les URL magiques ou bien en activant le mode HTML. Avec le mode HTML, les BBCodes, smilies et URL magiques ne fonctionneront pas, mais vous pouvez utiliser toute syntaxe HTML valide. Veuillez prendre note que ce contenu sera intégré dans un template HTML existant. Ainsi, nul besoin d’insérer les tags DOCTYPE, HTML, BODY ou HEAD. Néanmoins, tous les autres tags de mise en forme peuvent être utilisés, incluant IFRAME, SCRIPT, STYLE, EMBED, VIDEO, etc.',
 	'ACP_PAGES_FORM_TEMPLATE'			=> 'Template de la page',
-	'ACP_PAGES_FORM_TEMPLATE_EXPLAIN'	=> 'Des templates de page personnalisés, nommés <samp>pages_*.html</samp> peuvent être ajoutés dans les dossiers <samp>styles/*/template</samp> de phpBB.',
+	'ACP_PAGES_FORM_TEMPLATE_EXPLAIN'	=> 'Les templates des page personnalisées, sont nommés <samp>pages_*.html</samp> et peuvent être ajoutés dans les dossiers <samp>styles/*/template</samp> de phpBB.',
 	'ACP_PAGES_FORM_TEMPLATE_SELECT'	=> 'Sélectionner un template',
 	'ACP_PAGES_FORM_ORDER'				=> 'Ordre de la page',
-	'ACP_PAGES_FORM_ORDER_EXPLAIN'		=> 'Les pages seront triées selon ce champ, permettant ainsi d’aider à organiser l’ordre dans lequel vous souhaitez que leurs liens apparaissent. Les nombres les plus petits seront affichés avant les grands nombres.',
+	'ACP_PAGES_FORM_ORDER_EXPLAIN'		=> 'Les pages seront triées selon ce champ, permettant ainsi d’organiser l’ordre dans lequel vous souhaitez que leurs liens apparaissent. Les nombres les plus petits seront affichés avant les grands nombres.',
 	'ACP_PAGES_FORM_LINKS'				=> 'Emplacements pour le lien de la page',
-	'ACP_PAGES_FORM_LINKS_EXPLAIN'		=> 'Sélectionnez un/plusieurs endroits où le lien de cette page apparaîtra. Utilisez CTRL+CLICK (ou CMD+CLICK sur Mac) pour (dé)sélectionner plus d’un item.',
-	'ACP_PAGES_FORM_DISPLAY'			=> 'Afficher la page',
-	'ACP_PAGES_FORM_DISPLAY_EXPLAIN'	=> 'Si défini sur non, la page ne sera pas accessible. (Note: les administrateurs auront encore accès à la page, leur permettant d’en avoir un aperçu pendant qu’ils la préparent)',
-	'ACP_PAGES_FORM_GUESTS'				=> 'Afficher la page aux invités',
-	'ACP_PAGES_FORM_GUESTS_EXPLAIN'		=> 'Si défini sur non, seuls les utiliateurs enregistrés pourront accéder à la page.',
+	'ACP_PAGES_FORM_LINKS_EXPLAIN'		=> 'Sélectionnez un ou plusieurs emplacements où le lien de cette page apparaîtra. Utilisez CTRL+CLICK (ou CMD + CLICK sur Mac) pour (dé)sélectionner plus d’un item.',
+	'ACP_PAGES_FORM_DISPLAY'			=> 'Afficher la page ',
+	'ACP_PAGES_FORM_DISPLAY_EXPLAIN'	=> 'Si défini sur non, la page ne sera pas accessible aux utilisateurs néanmoins les administrateurs y auront accès, leur permettant d’avoir un aperçu pendant la création.',
+	'ACP_PAGES_FORM_GUESTS'				=> 'Afficher la page aux invités ',
```

Comment:
This should still be removed.
The template file uses `L_COLON` for the colon, which already contains the space in the french version.